src/pages/signUp/hook/useSignUpForm.ts

```
@@ -0,0 +1,87 @@
+  };
+  const passwordValue = watch('password');
+  const onChangePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue('password', e.target.value, { shouldValidate: true, shouldDirty: true });
+  };
+  const nameValue = watch('name');
+  const onChangeName = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValue('name', e.target.value.trim(), { shouldValidate: true, shouldDirty: true });
+  };
+  const handleSubmitWithoutEmailAuth = async () => {
+    const isValid = await trigger(['email', 'password', 'name']);
+    const onSubmit = (data: Pick<SignUpFormValues, 'email' | 'password' | 'name'>) => {
```

ESLint

[error] 'data' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)